File: tests/Generator/resource/rust/complex/any_property_type.rs
use serde::{Serialize, Deserialize};
use property_type::PropertyType;

// Represents an any value which allows any kind of value
#[derive(Serialize, Deserialize)]
pub struct AnyPropertyType {
    #[serde(rename = "deprecated")]
    deprecated: Option<bool>,

    #[serde(rename = "description")]
    description: Option<String>,

    #[serde(rename = "nullable")]
    nullable: Option<bool>,

    #[serde(rename = "type")]
    _type: Option<String>,

}

File: tests/Generator/resource/python/complex/any_property_type.py
from pydantic import BaseModel, Field, GetCoreSchemaHandler, Tag
from pydantic_core import CoreSchema, core_schema
from typing import Any, Dict, Generic, List, Optional, TypeVar, Annotated, Union, Literal
from .property_type import PropertyType


# Represents an any value which allows any kind of value
class AnyPropertyType(PropertyType):
    type: Literal["any"] = Field(alias="type")
    pass


